pcrypt.py

```diff
@@ -216,7 +216,7 @@ def read_password(show_password_in_prompt):
 
     def should_we_stop():
         return input(
-            '{white}Continue? [{yellow}y{reset}{white}/{reset}{yellow}n{reset}{white}]: {reset}'
+            '{white}Continue? [{yellow}y{reset}{white}/{reset}{yellow}N{reset}{white}]: {reset}'
```

Comment:
Hmmmm. So the `y` should be `Y` too. Am I right?

Reply:
```y``` is not default. When we type nothing or everything except ```y``` that means no (```n```) so ```y```(not Capital) is correct.

Reply:
I see. Thank you for clarifying.

Reply:
You're welcome :blush:.